internal/db/merge.go

```diff
@@ -58,24 +60,28 @@ func (db *db) executeMerge(ctx context.Context, dagMerge event.Merge) error {
 		return err
 	}
 
-	mt, err := getHeadsAsMergeTarget(ctx, txn, dsKey)
+	mt, err := getHeadsAsMergeTarget(ctx, txn, dsKey.WithFieldId(core.COMPOSITE_NAMESPACE))
 	if err != nil {
 		return err
 	}
 
-	err = mp.loadComposites(ctx, dagMerge.Cid, mt)
+	err = mp.loadBlocks(ctx, dagMerge.Cid, mt, false)
```

Comment:
question: why the name change from composites to blocks now?

Reply:
https://github.com/sourcenetwork/defradb/pull/2891#discussion_r1716463753